bindings/matrix-sdk-ffi/src/room_list.rs

```diff
@@ -417,13 +410,60 @@ impl RoomListDynamicEntriesController {
 
 #[derive(uniffi::Enum)]
 pub enum RoomListEntriesDynamicFilterKind {
-    All,
-    AllNonLeft,
+    All { filters: Vec<RoomListEntriesDynamicFilterKind> },
+    Any { filters: Vec<RoomListEntriesDynamicFilterKind> },
+    NonLeft,
+    Unread,
+    Category { expect: RoomListFilterCategory },
     None,
     NormalizedMatchRoomName { pattern: String },
     FuzzyMatchRoomName { pattern: String },
 }
 
+#[derive(uniffi::Enum)]
+pub enum RoomListFilterCategory {
+    Group,
```

Comment:
People is plural, so maybe Group should be Groups?

Reply:
Well yeah, I had the same thought :-D! A room contains a group, or multiple people, that's my reasoning.

Reply:
But rooms inside People generally contain a person, right? (If you exclude yourself...)

Element Web has "People" and "Rooms". But I am happy with whichever you prefer.